libs/lwt_platform/dune

```diff
@@ -0,0 +1,26 @@
+;; What is this? And why are we doing it?
```

Comment:
haha, double semicolon, Emacs influenced!

Reply:
Wait is it not double semicolon 😢 Emacs has ruined me